### PR TITLE
Normalizator and SDK update

### PR DESCRIPTION
**Pull request by mcuadros**
This a PR updating the SDK and configuring the normalizer package to work and generated the UAST.

Requires https://github.com/bblfsh/sdk/pull/205

### CODE REVIEW

File: driver/normalizer/tonode.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		"name": true,
+		"text": true, // for comments
+	},
+	// PHP AST includes a map called attributes with several properties, should
+	// be ignored, otherwise fake nodes are created.
+	IsNode: func(v map[string]interface{}) bool {
+		_, ok := v["nodeType"]
+		return ok
+	},
+	// The parse returns multiple nodes instead of a single node, a fake node
```
Comment (abeaumont) [inline driver/normalizer/tonode.go]: s/parse/parser/

File: driver/normalizer/tonode.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	},
+	Modifier: func(n map[string]interface{}) error {
+		// Sometimes the name if includes namespaces is given as an array in
+		// several parts. The parts are imploded into the name key.
+		if parts, ok := n["parts"].([]interface{}); ok {
+			n["name"] = sliceInterfaceToString(parts, "\\")
+			delete(n, "parts")
+		}
+
+		// In comments the positions doesn't follow the same schema of the
```
Comment (abeaumont) [inline driver/normalizer/tonode.go]: * s/In comments the positions doesn't/Positions in comments don't/
* s/of/as/
* s/please/place/

File: driver/normalizer/tonode.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		// other nodes, the position info is moved to the same please.
+		if pos, ok := n["filePos"].(float64); ok {
```
Comment (abeaumont) [inline driver/normalizer/tonode.go]: why float instead of int?

Comment (mcuadros) [reply driver/normalizer/tonode.go]: any number coming from a JSON decoded in a interface{} is a float64

File: driver/normalizer/annotation.go [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+import (
+	"github.com/bblfsh/sdk/uast/transformer/positioner"
+	"gopkg.in/bblfsh/sdk.v1/uast"
+	. "gopkg.in/bblfsh/sdk.v1/uast/ann"
+	"gopkg.in/bblfsh/sdk.v1/uast/transformer"
+	"gopkg.in/bblfsh/sdk.v1/uast/transformer/annotatter"
+)
+
+// Transformers is the of list `transformer.Transfomer` to apply to a UAST, to
```
Comment (juanjux) [inline driver/normalizer/annotation.go]: "Transformers is the list of ..."

File: driver/normalizer/tonode.go [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+	OnToNode: func(n interface{}) (interface{}, error) {
+		return map[string]interface{}{
+			"root": map[string]interface{}{
+				"nodeType": "File",
+				"children": n,
+			},
+		}, nil
+	},
+	Modifier: func(n map[string]interface{}) error {
+		// Sometimes the name if includes namespaces is given as an array in
```
Comment (juanjux) [inline driver/normalizer/tonode.go]: "Sometimes, if the name includes namespaces, it's given as..."

**Comment by juanjux** [comment]
Need to rebase with the latest SDK and update the managed files.

**Comment by juanjux** [comment]
CI is failing, looks like the image is not being built or is built with another name.

**Pull request merged**